README.md

```diff
@@ -145,3 +143,78 @@ isn't a valid chess move.
     0-1     -> black wins
     1-0     -> white wins
     1/2-1/2 -> draw
+
+## Configuration (for Spanish grammar)
+The parser can be configured to accept spanish grammar.
+
+    const options = {
+        language: 'es'
+        }
+    };
+    const parser = new ChessNLP(options);
+
+You can use both, aliases and language configurations:
+
+    const options = {
+        aliases: {
+            knight: ['horse', 'jumper'],
+            rook: ['tower'],
+            a: ['alpha'],
+            b: ['beta', 'bravo'],
+            c: ['charlie'],
+            1: ['i'],
+            2: ['ii'],
+            3: ['iii']
+        },
+        language: 'es'
+        }
+    };
+    
+    const parser = new ChessNLP(options);
+
+Language are mutually exclusive, if you set spanish, you can't parse english grammar and viceversa.
+Default language is english.
+
+#### sanToText(san) / fromSAN(san) examples (spanish grammar)
+
+    e4      -> e4
+    hxg2    -> h por g2
+    axb8=Q  -> a por b8 igual dama
```

Comment:
Is "igual" really the most natural way to say this in Spanish? I'm not a native speaker, but maybe "promociona a dama" or "corona con dama" would be a more natural way to describe the move in spoken language. "igual" sounds like you're just reading the algebraic notation out loud. (I see that "promociona" is included in the grammar file.)